instrumentation/runtime/runtime.go

```diff
@@ -26,18 +26,42 @@ import (
 
 // Runtime reports the work-in-progress conventional runtime metrics specified by OpenTelemetry
 type runtime struct {
-	meter    metric.Meter
-	interval time.Duration
+	meter              metric.Meter
+	minGCStatsInterval time.Duration
+}
+
+// Option supports configuring optional settings for runtime metrics.
+type Option func(*runtime)
+
+// DefaultMinimumGCStatsInterval is the default minimum interval
+// between calls to runtime.ReadMemStats().  Use the
+// WithMinimumGCStatsInterval() option to modify this setting in
+// Start().
+const DefaultMinimumGCStatsInterval time.Duration = 15 * time.Second
+
+// WithMinimumGCStatsInterval sets a minimum interval between calls to
+// runtime.ReadMemStats(), which is a relatively expensive call to make
+// frequently.  `d` values less than 0 will be disregarded silently.
+func WithMinimumGCStatsInterval(d time.Duration) Option {
+	return func(r *runtime) {
+		if d >= 0 {
+			r.minGCStatsInterval = d
+		}
+	}
 }
 
 // New returns Runtime, a structure for reporting Go runtime metrics
 // interval is used to limit how often to invoke Go runtime.ReadMemStats() to obtain metric data.
 // If the metric SDK attempts to observe MemStats-derived instruments more frequently than the
 // interval, a cached value will be used.
-func Start(meter metric.Meter, interval time.Duration) error {
+func Start(provider metric.Provider, opts ...Option) error {
```

Comment:
The `Provider` should be passed as an option with the default, if none are passed, being the `global.MeterProvider` (https://github.com/open-telemetry/opentelemetry-go-contrib/issues/213).

Reply:
Fixed.